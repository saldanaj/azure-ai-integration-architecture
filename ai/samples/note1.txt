Patient: Sarah Connor (P123)
Encounter: E456 | Discharge Date: 2024-02-12
Primary Diagnosis: Acute decompensated heart failure.
Hospital Course: Improved with IV diuretics, transitioned to oral medications.

Follow-up Instructions:
1. Labs: Obtain a basic metabolic panel in 3 days to monitor renal function and potassium after starting lisinopril.
2. Visit: Schedule a cardiology follow-up within 7 days to assess volume status and titrate meds.
3. Medication: Nursing team to call the patient in 48 hours to reinforce low-sodium diet and confirm medication adherence.

Discharge Medications: Furosemide, Lisinopril, Spironolactone.
MRN: 555443
