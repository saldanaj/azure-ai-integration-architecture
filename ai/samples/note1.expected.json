{
  "followUps": [
    {
      "category": "lab",
      "title": "Order basic metabolic panel to monitor renal function and potassium after lisinopril start",
      "dueDate": "2024-02-15",
      "priority": "normal"
    },
    {
      "category": "visit",
      "title": "Schedule cardiology follow-up visit to reassess volume status and adjust medications",
      "dueDate": "2024-02-19",
      "priority": "normal"
    },
    {
      "category": "med",
      "title": "Conduct nursing phone call to reinforce low-sodium diet and confirm medication adherence",
      "dueDate": "2024-02-14",
      "priority": "normal"
    }
  ]
}
